pkg/server/apiserver/apiserver.go

```diff
@@ -101,7 +102,7 @@ func (c completedConfig) New() (*TiltServer, error) {
 	}
 
 	// Add new APIs through inserting into APIs
-	apiGroups, err := buildAPIGroupInfos(c.ExtraConfig.Scheme, c.ExtraConfig.Codecs, c.ExtraConfig.APIs, c.GenericConfig.RESTOptionsGetter)
+	apiGroups, err := buildAPIGroupInfos(c.ExtraConfig.Scheme, c.ExtraConfig.Codecs, c.ExtraConfig.APIs, c.GenericConfig.RESTOptionsGetter, c.ExtraConfig.ParameterCodec)
```

Comment:
Custom ParameterCodec allows to define kind- and sub-resource-specific parameters for REST requests.